test/prob/binomial/binomial_cdf_test.hpp

```diff
@@ -52,10 +52,10 @@ class AgradCdfBinomial : public AgradCdfTest {
     using std::exp;
     using std::log;
 
-    stan::return_type_t<T_prob> cdf(1);
+    stan::return_type_t<T_prob> cdf(0);
 
     for (int i = 0; i <= n; i++) {
-      cdf *= binomial_coefficient<double>(N, i)
+      cdf += binomial_coefficient<double>(N, i)
```

Comment:
This code is the same as the log cdf code without the log: https://github.com/stan-dev/math/blob/develop/test/prob/binomial/binomial_cdf_log_test.hpp